manifests/spel-minimal-centos-7-hvm.manifest.txt

```diff
@@ -1,485 +1,36 @@
-python-iniparse-0.4-9.el7.noarch
-python-ipaddress-1.0.16-2.el7.noarch
-python-IPy-0.75-6.el7.noarch
-python-javapackages-3.4.1-11.el7.noarch
-python-jinja2-2.7.2-2.el7.noarch
-python-jsonpatch-1.2-4.el7.noarch
-python-jsonpointer-1.9-2.el7.noarch
-python-kitchen-1.1.1-5.el7.noarch
-python-libs-2.7.5-77.el7_6.x86_64
-python-linux-procfs-0.4.9-4.el7.noarch
-python-lockfile-0.9.1-4.el7.centos.noarch
-python-lxml-3.2.1-4.el7.x86_64
-python-markupsafe-0.11-10.el7.x86_64
-python-perf-3.10.0-957.10.1.el7.x86_64
-python-prettytable-0.7.2-3.el7.noarch
-python-pycurl-7.19.0-19.el7.x86_64
-python-pyudev-0.15-9.el7.noarch
-python-requests-2.6.0-1.el7_1.noarch
-python-schedutils-0.4-6.el7.x86_64
-python-setuptools-0.9.8-7.el7.noarch
-python-six-1.9.0-2.el7.noarch
-python-slip-0.4.0-4.el7.noarch
-python-slip-dbus-0.4.0-4.el7.noarch
-python-urlgrabber-3.10-9.el7.noarch
-python-urllib3-1.10.2-5.el7.noarch
-pyxattr-0.5.1-5.el7.x86_64
-PyYAML-3.10-11.el7.x86_64
-qrencode-libs-3.4.1-3.el7.x86_64
-rdma-core-17.2-3.el7.x86_64
-readline-6.2-10.el7.x86_64
-rootfiles-8.1-11.el7.noarch
-rpm-4.11.3-35.el7.x86_64
-rpm-build-libs-4.11.3-35.el7.x86_64
-rpm-libs-4.11.3-35.el7.x86_64
-rpm-python-4.11.3-35.el7.x86_64
-rsync-3.1.2-4.el7.x86_64
-rsyslog-8.24.0-34.el7.x86_64
-ruby-2.0.0.648-34.el7_6.x86_64
-rubygem-bigdecimal-1.2.0-34.el7_6.x86_64
-rubygem-io-console-0.4.2-34.el7_6.x86_64
-rubygem-json-1.7.7-34.el7_6.x86_64
-rubygem-psych-2.0.0-34.el7_6.x86_64
-rubygem-rdoc-4.0.0-34.el7_6.noarch
-rubygems-2.0.14.1-34.el7_6.noarch
-ruby-irb-2.0.0.648-34.el7_6.noarch
-ruby-libs-2.0.0.648-34.el7_6.x86_64
-sed-4.2.2-5.el7.x86_64
-selinux-policy-3.13.1-229.el7_6.9.noarch
-selinux-policy-targeted-3.13.1-229.el7_6.9.noarch
-setools-libs-3.3.8-4.el7.x86_64
-setup-2.8.71-10.el7.noarch
-sg3_utils-1.37-17.el7.x86_64
-sg3_utils-libs-1.37-17.el7.x86_64
-shadow-utils-4.1.5.1-25.el7_6.1.x86_64
-shared-mime-info-1.8-4.el7.x86_64
-slang-2.2.4-11.el7.x86_64
-snappy-1.1.0-3.el7.x86_64
-sqlite-3.7.17-8.el7.x86_64
-sudo-1.8.23-3.el7.x86_64
-systemd-219-62.el7_6.5.x86_64
-systemd-libs-219-62.el7_6.5.x86_64
-systemd-sysv-219-62.el7_6.5.x86_64
-sysvinit-tools-2.88-14.dsf.el7.x86_64
-tar-1.26-35.el7.x86_64
-tcp_wrappers-libs-7.6-77.el7.x86_64
-teamd-1.27-5.el7.x86_64
-ttmkfdir-3.0.9-42.el7.x86_64
-tuned-2.10.0-6.el7_6.3.noarch
-tzdata-2019a-1.el7.noarch
-tzdata-java-2019a-1.el7.noarch
-ustr-1.0.4-16.el7.x86_64
-util-linux-2.23.2-59.el7_6.1.x86_64
-vim-common-7.4.160-5.el7.x86_64
-vim-filesystem-7.4.160-5.el7.x86_64
-vim-minimal-7.4.160-5.el7.x86_64
-virt-what-1.18-4.el7.x86_64
-wget-1.14-18.el7.x86_64
-which-2.20-7.el7.x86_64
-wpa_supplicant-2.6-12.el7.x86_64
-xfsprogs-4.5.0-19.el7_6.x86_64
-xorg-x11-fonts-Type1-7.5-9.el7.noarch
-xorg-x11-font-utils-7.5-21.el7.x86_64
-xz-5.2.2-1.el7.x86_64
-xz-libs-5.2.2-1.el7.x86_64
-yum-3.4.3-161.el7.centos.noarch
-yum-metadata-parser-1.1.4-10.el7.x86_64
-yum-plugin-fastestmirror-1.1.31-50.el7.noarch
-yum-utils-1.1.31-50.el7.noarch
-zlib-1.2.7-18.el7.x86_64
+
```

Comment:
none of the manifest files are correct...

Reply:
ok, so wget didn't work.  lol.  ok, I got the windows mounting work and got the right manifests downloaded